parity-clib-python-example/parity.h

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 Parity Technologies (UK) Ltd.
```

Comment:
Same remark as the other `parity.h`.
It would be better not to duplicate files.